src/main/java/core/basesyntax/Car.java

```diff
@@ -1,5 +1,22 @@
 package core.basesyntax;
 
-public class Car {
+public class Car implements Cloneable {
+    private Engine engine;
+    private Color color;
+    private double weight;
+    private String model;
+    private boolean isnew;
+
+    private Car(Engine engine, Color color, double weight, String model, boolean isnew) {
+        this.engine = engine.clone();
```

Comment:
we don't need to make a cope when we are creating Car, but when we are cloning it

Reply:
Haven`t sure understood you clearly, but done

Reply:
look closely to the second part of comment. We should clone Engine when we cloning a car